src/modules/jcms_rest/src/EventSubscriber/ExceptionElifeJsonSubscriber.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+namespace Drupal\jcms_rest\EventSubscriber;
+
+use Drupal\Core\EventSubscriber\ExceptionJsonSubscriber;
+
+/**
+ * Handle eLife JSON exceptions the same as JSON exceptions.
+ */
+class ExceptionElifeJsonSubscriber extends ExceptionJsonSubscriber {
```

Comment:
what does this class do? If there is an exception in the resource class, where can I see it? (in the response or in some log)